web/elm/src/Build.elm

```diff
@@ -107,7 +107,7 @@ init flags page =
                 , previousKeyPress = Nothing
                 , previousTriggerBuildByKey = False
                 , showHelp = False
-                , hash = flags.hash
```

Comment:
I'm going to admit to being kinda really steamed that we have to store this... but there's no way to pass it directly to the Build.Output, is there. :(

Reply:
The highlight, not the hash.

Reply:
I think the Right Thing to do here is merge all the `Build`-related models, potentially using extensible records to scope it in different modules... I guess we should poke around a bit more on this since we're touching it now anyway.

Reply:
Yeah maybe. There's a bunch of them, though - it might make sense to do that as a separate, second pr.